mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -491,12 +511,16 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
 
     private fun queryServerProperties() {
         propsUpdateHandle?.cancel()
+        retryJob?.cancel(CancellationException("queryServerProperties() was called"))
         val successCb: (ServerProperties) -> Unit = { props ->
             serverProperties = props
             updateSitemapAndHabPanelDrawerItems()
             if (props.sitemaps.isEmpty()) {
                 Log.e(TAG, "openHAB returned empty Sitemap list")
                 controller.indicateServerCommunicationFailure(getString(R.string.error_empty_sitemap_list))
+                scheduleRetry {
+                    retryServerPropertyQuery()
+                }
```

Comment:
Does a retry really help here?

Reply:
I don't think this error is shown at all, because the server always returns a sitemap based on things.
But let's say the app connects to the server in the same moment the server reloads the sitemap definitions (and the default sitemap is not available). In this case a retry helps.